analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/ObjectsShouldNotBeDisposedMoreThanOnceTest.cs

```diff
@@ -59,6 +65,16 @@ public class ObjectsShouldNotBeDisposedMoreThanOnceTest
 
 #if NET
 
+        [DataTestMethod]
+        [DataRow(ProjectType.Product)]
+        [DataRow(ProjectType.Test)]
+        public void ObjectsShouldNotBeDisposedMoreThanOnce_Roslyn_VB(ProjectType projectType) =>
```

Comment:
Do we care also to split VB to C#8 and other versions?

Reply:
My hands went faster than my brain once again: `DisposeASync` is available only in .NET core. Should I split the tests in .NET and .NET framework?
I would say no - but why not ask.

Reply:
Main tests should be outside `#if NET` statement. So you should either delete DisposeAsync, because it's already tested on C# side (it's in the base class anyway), or split it and add `.Net.cs.` suffix

Reply:
It's not in the base class anymore - and if it's not tested it drops the coverage - so I'll split the tests.